packages/cli/src/cmds/beacon/initPeerIdAndEnr.ts

```diff
@@ -154,11 +155,11 @@ export async function initPrivateKeyAndEnr(
     let privateKey: PrivateKey;
     let enr: SignableENR;
 
-    // attempt to read stored peer id
+    // attempt to read stored private key
     try {
       privateKey = readPrivateKey(peerIdFile);
     } catch (_e) {
-      logger.warn("Unable to read peerIdFile, creating a new peer id");
+      logger.warn("Unable to read peerIdFile, creating a new private key");
```

Comment:
creating a new private key implies a new peer id, I would probably just keep the log as is